.github/pytorch-probot.yml

```diff
@@ -23,6 +23,7 @@ ciflow_push_tags:
 - ciflow/xpu
 - ciflow/torchbench
 - ciflow/autoformat
+- ciflow/win-arm64
```

Comment:
Why are you adding this one? `ciflow/` labels are usually reserved for CI, for CD everything should be triggered by `ciflow/binaries`